src/dfind/dfind.c

```diff
@@ -516,13 +538,20 @@ int main (int argc, char** argv)
     	        exit(1);
             }
     	    break;
-
+#ifdef DAOS_SUPPORT
+            case 'X':
+                daos_args->dfs_prefix = MFU_STRDUP(optarg);
+                break;
+#endif
```

Comment:
@daltonbohning its too bad we still have to keep adding this daos_prefix to more tools since we will be eventually getting rid of it

Reply:
@dsikich Yeah :/
I wonder if it's okay to just not add it? Do you know when we plan to have support such that we won't need it?

Reply:
@daltonbohning I think it is supposed to be in the 2.0 release but not 100 percent sure. It is probably not necessary to have it in all of these since there is a way to copy subsets using the daos:// direct format, and we will have to go back and remove it as part of the interface in the future.

Reply:
@dsikich Okay, I do agree with that. I removed prefix from drm, dwalk, dfind. Now we only have it in dcp, dsync, dcmp (it was already in those).
Worst case, someone will specifically request it and we can add it. But like you said, the direct format allows it as well (and is easier IMO)

Reply:
@daltonbohning yeah, I think using the prefix is awkward.